### PR TITLE
feat: add coverage for node range indexes 

### PR DESCRIPTION
**Pull request by fbiville**
This adds coverage for the range indexes of the node schema section of the spec.

### CODE REVIEW

**Comment by neo4j-connectors** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>PR Title ' feat: add coverage for node range indexes ': subject may not be empty, type may not be empty</td>
    </tr>
  </tbody>
</table>



> Pull request title should follow [Conventional commits](https://cheatography.com/albelop/cheat-sheets/conventional-commits).
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against badc507d6bd210b48b65833c019021e3fffa639f
</p>

File: src/test/java/org/neo4j/importer/v1/ImportSpecificationDeserializerNodeTargetTest.java [hunk 1]
```diff
@@ -4186,4 +4186,621 @@ public void fails_if_node_schema_key_constraint_options_is_wrongly_typed() {
...
+                                    "name": "a-target",
+                                    "source": "a-source",
+                                    "write_mode": "merge",
+                                    "labels": ["Label"],
+                                    "properties": [
+                                        {"source_field": "field", "target_property": "property"}
+                                    ],
+                                    "schema": {
+                                        "range_indexes": [
+                                            {"name": "a key constraint", "properties": ["property"]}
```
Comment (ali-ince) [inline src/test/java/org/neo4j/importer/v1/ImportSpecificationDeserializerNodeTargetTest.java:4443]: nit: do you want to change the name here and the rest of the tests?

**Review by ali-ince — APPROVED** [review]

File: src/test/java/org/neo4j/importer/v1/ImportSpecificationDeserializerNodeTargetTest.java (hunk 1 above)
Comment (fbiville) [reply src/test/java/org/neo4j/importer/v1/ImportSpecificationDeserializerNodeTargetTest.java:4443]: yep, fix on the way

Comment (fbiville) [reply src/test/java/org/neo4j/importer/v1/ImportSpecificationDeserializerNodeTargetTest.java]: fixed in aa80352a629888c54f763480cca12f8523f0cae8

**Review by ali-ince — APPROVED** [review]

**Pull request merged**